### DISCUSSION TITLE
Issue to increase interaction in GitHub.

### DISCUSSION THREAD

**Discussion by Lazeeez** [Lazeeez/HacktoberFest-21#53] [Ideas]
Github majorly revolves around the interactivity of the repository (stars and forks and commits, etc.). comment here if you wanna exchange stars and forks to increase interactivity.

**Comment by itsDigvijaysing** [comment]
Hello

**Comment by Lazeeez** [comment]
> Hello

Hi,
lets exchanges stars for the repositories first and follow each other.

**Comment by itsDigvijaysing** [comment]
Done

**Comment by Lazeeez** [comment]
@RoyalTechie good good, now make sure you spread it out. The more the better.

**Comment by Lazeeez** [comment]
@meks-0202 are you interested?

**Comment by meks-0202** [comment]
Yeah sure.

**Comment by Lazeeez** [comment]
@RoyalTechie @AnAi05 and @meks-0202 make sure you all follow each other and try to expand this as more as possible to your friends, colleagues, peers, co-workers... the more people the better

**Comment by Lazeeez** [comment]
@salonishukla206 are you interested?

**Comment by prokan468** [comment]
@Lazeeez Sure! We should do that! Following you right away :)

**Comment by Lazeeez** [comment]
> @Lazeeez Sure! We should do that! Following you right away :)

Sure man, same here.

**Comment by Lazeeez** [comment]
@AnAi05 @RoyalTechie @meks-0202 @prokan468 make sure you all follow each other, thanks.

PS: Try to add more people here. Link this issue wherever you can.

**Comment by Nifalch** [comment]
Hi
